news/2025/2025-08-22-beatmap-spotlights-summer-2025.md

```diff
@@ -0,0 +1,216 @@
+---
+layout: post
+title: "Beatmap Spotlights: Summer 2025"
+date: 2025-08-22 18:00:00 +0000
+---
+
+Beatmap Spotlights are back for Summer 2025 Season! Check out the fresh selection of beatmaps and compete for highest placement in the seasonal rankings!
+
+![](/wiki/shared/news/2025-08-22-beatmap-spotlights-summer-2025/banner.jpg)
+
+Even though it's August already, the sun is still burning outside brightly so what else there's to enjoy other than a full sweat-on run through best beatmaps of last months, carefully selected by a designated Beatmap Spotlights Curators team!
```

Comment:
sun is still burning **brightly outside**
or
sun is still **outside burning** brightly

what else **is there** to enjoy

through **the** best beatmaps of **the** last **few** months

Beatmap Spotlights Curator team
or
team of Beatmap Spotlights Curators